test/ci/launch.yml

```diff
@@ -0,0 +1,80 @@
+---
+- hosts: localhost
+  connection: local
+  gather_facts: false
+  tasks:
+    - include_vars: "{{ item }}"
```

Comment:
Are we saying that there should be a vars.yaml on the host that is running ansible that supplies some variables?  This file is provided by CI?  Let's add a comment if so.